spec/yaml/swiftnav/sbp/acquisition.yaml

```diff
@@ -70,3 +70,50 @@ definitions:
             desc: |
               PRN-1 identifier of the satellite signal for which
               acquisition was attempted
+
+ - MSG_ACQ_SV_PROFILE:
+    id: 0x001E
+    short_desc: Reacq perfomance measurement and debug
+    desc: |
+      The message describes SV profile during acquisition time.
+      The message is useful to debug and measure the performance of
+      the new (re-)acq design.
+    public: true
+    fields:
+        - N:
```

Comment:
How does this message get laid out? This field refers to a count of SV's to be included in the message - yet there's no subsequent collection of anything. There are other messages in SBP to reference where there are collections of message objects, e.g., MSG_OBS, MSG_TRACKING_IQ, MSG_TRACKING_STATE.